pom.xml

```diff
@@ -59,6 +59,7 @@
     <test-integration_add-source-phase>pre-integration-test</test-integration_add-source-phase>
     <maven-checkstyle-plugin.version>2.17</maven-checkstyle-plugin.version>
     <docjs.config.file></docjs.config.file>
+    <jsdoc.local>true</jsdoc.local>
```

Comment:
I couldn't help noticing the different prefixes used for the same namespace: `docjs` and `jsdoc`.